### PR TITLE
CHG11111: Update to Gymky caption file

### LINKED ISSUE #5: Activity 1: Your First Caption
Using the API, I have just created a file with a random octocat for you to caption. In this activity, we will learn how to make changes to files in our GitHub repository.

- [ ] @beardofedu
- [ ] @satyapenta
- [ ] @FShenoda
- [ ] @peterhugger
- [ ] @BinduTuli0609
- [ ] @a00125970
- [ ] @NowshadAzimi
- [ ] @wsmith2
- [ ] @chinrung
- [ ] @RonkasPSEG
- [ ] @designMoreWeb
- [ ] @Gymky
- [ ] @MatthewSchlosserPSEG
- [ ] @derekacevedo
- [ ] @Shdesai36
- [ ] @shreykhu
- [ ] @TerHano
- [ ] @arch02
- [ ] @awhite575
- [ ] @wsmith2
- [ ] @beardofedu
- [ ] @Mike-the-powerful
- [ ] @XinyuXiong
- [ ] @beardofedu
- [ ] @beardofedu

### PR DESCRIPTION
**Pull request by Gymky**
This was a change to the '_post' file, it update the caption to be more inline with the image provided.

closes #5

### CODE REVIEW

File: _posts/0000-01-12-Gymky.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Changing caption of my cool octocat
```
Comment (beardofedu) [inline _posts/0000-01-12-Gymky.md]: Perhaps mention something Rainbow Dash would say? No idea what their catchphrase is 🤷‍♂

File: _posts/0000-01-12-Gymky.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: slide
...
 
 ![cooler-octocat](https://octodex.github.com/images/twenty-percent-cooler-octocat.png)
```
Comment (beardofedu) [inline _posts/0000-01-12-Gymky.md]: If you don't like your image, you can always pick a new one from [here](https://octodex.github.com/)

File: _posts/0000-01-12-Gymky.md [hunk 3]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
```
Comment (beardofedu) [inline _posts/0000-01-12-Gymky.md]: Maybe change your slide title to: 

```suggestion
title: "Welcome to Gymky's slide!"
```

File: _posts/0000-01-12-Gymky.md (hunk 2 above)
Comment (Gymky) [reply _posts/0000-01-12-Gymky.md]: image was good, just didn't know what to name it.

**Pull request merged**